server/Cargo.toml

```diff
@@ -14,17 +14,22 @@ path = "src/lib/lib.rs"
 [[bin]]
 name = "propolis-server"
 path = "src/main.rs"
+# Both the lib and bin share the same name and so only one may be documented
+doc = false
 
 [dependencies]
 anyhow = "1.0"
 # dropshot = "0.6"
 dropshot = { git = "https://github.com/oxidecomputer/dropshot", branch = "main" }
 futures = "0.3"
+hyper =  "0.14"
 thiserror = "1.0"
 tokio = { version = "1", features = ["full"] }
+tokio-tungstenite = "0.14"
```

Comment:
There was also `async-tungstenite` as an option. I ultimately chose `tokio-tungstentite` because we're an all tokio stack right now and so we don't need this crate to be runtime-agnostic. A nice plus is that the `tokio-tungstenite` and base `tungstenite` crates are maintained by the same folks.

`hyper-tungstenite` is another crate that fits the niche and is pretty ideal except for the fact it wants to take the hyper `Request<Body>` object by value which isn't possible via a dropshot handler currently.